rspace/src/main/scala/coop/rchain/rspace/nextgenrspace/history/HistoryInstances.scala

```diff
@@ -11,7 +11,7 @@ import scala.collection.concurrent.TrieMap
 
 object HistoryInstances {
 
-  val MalformedTrieError = new RuntimeException("malformed trie")
+  def MalformedTrieError = new RuntimeException("malformed trie")
```

Comment:
`val` was correct. Why create a new instance of `RuntimeException` each time `MalformedTrieError` is dereferenced?

Reply:
to have a stacktrace from the spot where it was thrown not where the val was created

Reply:
Right! I missed that